### DISCUSSION TITLE
JUnit Reporter Fails to load

### DISCUSSION THREAD

**Discussion by oravecz-jpmc** [vitest-dev/vitest#954] [General]
I have a complex monorepo, so I'm not sure if this is just my environment or if others are seeing it. If others are having this problem, I'll work up a stripped down example.

Using Vitest v6.0.1

Config (`vite.config.ts`)
```
  test: {
    environment: 'jsdom',
    coverage: {
      reporter: ['lcov', 'junit'],
      reportsDirectory: 'test-reports',
    },
  },
```

Error
```
Error: Cannot find module 'junit'
Require stack:
- /Users/foo/node_modules/istanbul-reports/index.js
- /Users/foo/node_modules/c8/lib/report.js
- /Users/foo/node_modules/vitest/dist/chunk-defaults.52b18365.js
 ❯ Function.Module._resolveFilename internal/modules/cjs/loader.js:902:15
 ❯ Function.Module._load internal/modules/cjs/loader.js:746:27
 ❯ Module.require internal/modules/cjs/loader.js:974:19
 ❯ require internal/modules/cjs/helpers.js:93:18
 ❯ Object.create ../../../node_modules/istanbul-reports/index.js:19:20
 ❯ Report.run ../../../node_modules/c8/lib/report.js:73:15
```

**Comment by Demivan** [comment]
You are adding junit as a coverage reporter.
Junit is a test reporter. Add it in `test.reporters` config and not in `test.coverage.reporter`.

**Reply by oravecz-jpmc** [reply]
Thanks, that was definitely the problem

**Reply by zhao-li** [reply]
This solved my issue as well.  I wish the this documentation (https://vitest.dev/config/#reporters) would be more explicit on the structure of the configurations.

Thank you for your help 🙏
